src/services/classification/index.ts

```diff
@@ -0,0 +1,36 @@
+import { axiosInstance } from '@/services/client/api-client'
+
+const CERTAINTY_THRESHOLD = 0.41
+const MAIN_SLUG_REGEX = /\/terms\/categories\/([a-z0-9\-]+)/
+const SUB_SLUG_REGEX =
+  /\/terms\/categories\/[a-z0-9\-]+\/sub_categories\/([a-z0-9\-]+)/
+
+export const getCategoryForDescription = async (description: string) => {
+  const axios = axiosInstance(process.env.NEXT_PUBLIC_BASE_URL_API)
+
+  let prediction = {
+    main: 'overig',
+    sub: 'overig',
+  }
+
+  try {
+    const response = await axios.post(`/signals/category/prediction`, {
+      text: description,
+    })
+
+    const mainCertainty = response.data.hoofdrubriek[1][0]
```

Comment:
moet hier nog een extra check op of hoofdrubriek aanwezig is met daarin een array die een array bevat?

Reply:
De API zou altijd hoofdrubriek en subrubriek teruggeven als de response OK is, dus ik denk dat dit wel goed zit.